### PR TITLE
Add SendMessage event

### PR DESCRIPTION
**Pull request by gsouquet**
Adds a `SendMessage` event to the collection of events sent to PostHog.

The parameters are based on the ones sent to Countly (view https://github.com/matrix-org/matrix-react-sdk/blob/5290afcc4c0a7c56b425f2784b86f3e729f5fc64/src/CountlyAnalytics.ts#L869), and adds a `threadId` string.

I was unsure how to express `msgtype` and `format` as those two fields have a set of values that we could enumerate, but they also allow any arbitrary string (unstable msgtype in case of an in-progress MSC for example)

### CODE REVIEW

File: schemas/SendMessage.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "type": "object",
+  "description": "Triggered when the user sends a message",
+  "properties": {
+    "eventName": {
+      "enum": ["SendMessage"]
+    },
+    "roomId": {
```
Comment (novocaine) [inline schemas/SendMessage.json:8]: Room Ids are PII, we won't send them as analytics data. What do you need this for?

File: schemas/SendMessage.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "type": "string"
+    },
+    "isEdit": {
+      "type": "boolean"
+    },
+    "isReply": {
+      "type": "boolean"
+    },
+    "msgType": {
+      "type": "string"
```
Comment (novocaine) [inline schemas/SendMessage.json:21]: should this be an enum?

File: schemas/SendMessage.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    },
+    "format": {
+      "type": "string"
```
Comment (novocaine) [inline schemas/SendMessage.json:24]: enum?

**Comment by novocaine** [comment]
Note that in https://docs.google.com/spreadsheets/d/1YHYibK7tfHbMvvlCAZEXSwPhCk9Yw69-0pvAO7ATv0A/edit#gid=557292659 its `SendMessage`

I guess we should standardise on present or past tense

**Comment by gsouquet** [comment]
Going to close this one as my event definiton is not compliant with our analytics policy and that I won't be needed this anymore

**Pull request closed without merging**